### ISSUE TITLE
Visual label is not provided for the Chart: A11y_.NET 9 template_.NET 9 template_Home_Labels or Instrunctions.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26232]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG3.3.2; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Labels or Instructions;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till checkboxes section.
4. Observe that Visual label is not provided for the Chart.

### Actual Result:
Visual label is not provided for the Chart.

### Expected Result:
Visual label should be provided for the Chart.

### User Impact:
Users will face difficulties in knowing the purpose of the Chart as visual label is not provided for the Chart.

### Attachment:
![Image](https://github.com/user-attachments/assets/f2aced88-8657-4910-bb51-8eb7687dab91)

**Comment by AnKushSingh05** [comment]
Issue is fixed, Hence closing the bug.

![Image](https://github.com/user-attachments/assets/13e78035-447e-4ff3-81a4-a55b9a2c1894)
